easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.16-GCC-10.2.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'VCFtools'
+version = '0.1.16'
+
+homepage = "https://vcftools.github.io"
+description = """The aim of VCFtools is to provide 
+ easily accessible methods for working with complex 
+ genetic variation data in the form of VCF files."""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/%(namelower)s/%(namelower)s/releases/download/v%(version)s/']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+checksums = ['dbfc774383c106b85043daa2c42568816aa6a7b4e6abc965eeea6c47dde914e3']
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
+    ('binutils', '2.35'),
```

Comment:
binutils is already part of the GCC toolchain (but not the GCCcore) so this is not necessary.

Also, does this use pkg-config? If so, we should add it in previous easyconfigs as well.

Reply:
removing Binutils, pkg-config was required. I build on Ubuntu and I always find the missing pkg-config issues.